src/index.ts

```diff
@@ -3,5 +3,6 @@ import Container from "./Container";
 import Provider from "./Provider";
 
 export * from "./types";
+export * from "./Provider";
```

Comment:
Well, i've searched and also opened a question on stackoverflow, i got a good direction on what is going wrong if we do: `import {interface} from 'a'` and `export {interface} `.

The discussion is on this Github issue: https://github.com/babel/babel-loader/issues/603

Basically, there are 2 options:

`export * from 'file'`

```
import Something {interface} from 'file';
export {Something};
export type InterfaceType = interface;
```

There is a 3rd one, but it hacky.

I prefered the first one because it is clean. If the second one is preferable, i can change!